src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3609,7 +3609,7 @@ private void recordRelocation(void* location, void* locationRW, void* target, us
 
                 default:
                     // Reloc points to something outside of the generated blocks
-                    var targetObject = HandleToObject((IntPtr)target);
+                    var targetObject = HandleToObject((void*)target);
```

Comment:
```suggestion
                    var targetObject = HandleToObject(target);
```
target is void* already